fecgov/openFEC
# [NO MERGE!!!]adding sched_a baseline walking list and walking script for reference

## Summary (required)
added the sched_a baseline walk script and endpoints list. Some data skewness analysis also added for reference
- Resolves #[_3618_]

_Include a summary of proposed changes._
those files are for research and reference only:
- the endpoints list file is sched_a.list
- run 'python sched_a_walk.py', the script will pick up the list and run through them
- data skewness(data distribution is not even and heavily tied to some argument values) is something we need to consider when we pick up the sched_a secondary argument list and test for performance
pls note: due to performance issue, the skewness files I provided here is based on a specific cycle/report year, it is for reference only
## How to test the changes locally
- check out the code
- run 'python sched_a_walk.py' in the baseline folder
-

## Impacted areas of the application
List general components of the application that this PR will affect:

-  remove 2-year restriction from sched_a 



## Related PRs
List related PRs against other branches:

branch | PR
------ | ------
fix/other_pr | [link]()
feature/other_pr | [link]()
